packages/ui/src/machines/authenticator/defaultServices.ts

```diff
@@ -60,6 +65,69 @@ export const defaultServices = {
 
   // Validation hooks for overriding
   async validateCustomSignUp(formData, touchData): Promise<ValidatorResult> {},
+  async validateFormPassword<Validator>(
+    formData,
+    touchData,
+    passwordSettings: PasswordSettings
+  ): Promise<ValidatorResult> {
+    const { password } = formData;
+
+    const { password: touched_password } = touchData;
+
+    /**
+     * If the password is not touched,
+     * or if the password settings are not set, we don't need to validate it.
+     */
+    if (!touched_password || !passwordSettings) return null;
```

Comment:
If their zero config doesn't have this in there, it should just skip

Reply:
Also if they haven't touched the password yet, it should skip, *same as checking only on blur*